albumentations/augmentations/functional.py

```diff
@@ -552,6 +552,46 @@ def bbox_random_crop(bbox, crop_height, crop_width, h_start, w_start, rows, cols
     return crop_bbox_by_coords(bbox, crop_coords, crop_height, crop_width, rows, cols)
 
 
+def bbox_rot90(bbox, factor, rows, cols):
+    """Rotates a bounding box by 90 degrees CCW (see np.rot90)
+
+    :param bbox:
+    :param n: Number of rotations [0;3]
+    :param rows:
+    :param cols:
+    :return:
+    """
```

Comment:
We are using Google Style docstrings (here is an example of this style - http://www.sphinx-doc.org/en/stable/ext/example_google.html)

So the dosctring should look something like this:
```
"""Rotates a bounding box by 90 degrees CCW (see np.rot90)

Args:
    n (int): Number of rotations [0;3]
"""
```
This is not exactly what Google Style recommends (because this docstring doesn't describe all parameters), but I think that it provides better readability.